### PR TITLE
Widget UI improved

### PR DESCRIPTION
**Pull request by SandroBoka**
Every widget has its own look now.
String localization added.
Colors are now shared everywhere by being in the Assets and having a dark and light theme.

### CODE REVIEW

File: WeatherApp/WeatherApp/App/Resources/Fonts/AppFonts.swift [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+extension Font {
+
+    static func dottedFont(size: Double) -> Font {
+        return Font.custom("NDOT45inspiredbyNOTHING", size: size)
+    }
+
+    static func notoSansFont(size: Double) -> Font {
+        return Font.custom("Noto Sans Mono", size: size)
+    }
```
Comment (dnikolic98) [inline WeatherApp/WeatherApp/App/Resources/Fonts/AppFonts.swift]: no need for return

File: WeatherApp/WeatherApp/App/Data/Repositories/WeatherModel.swift [hunk 2]
```diff
@@ -7,6 +7,7 @@ struct WeatherModel {
     let description: String
     let humidity: Int
     let speed: Double
+    let deg: Int
```
Comment (dnikolic98) [inline WeatherApp/WeatherApp/App/Data/Repositories/WeatherModel.swift]: degrees

File: WeatherApp/WeatherApp/Feature/Components/WindWidgetView.swift [hunk 3]
```diff
@@ -3,36 +3,34 @@ import SwiftUI
 struct WindWidgetView: View {
     var title: String
     var value: String
-    var deg: Double
+    var deg: Int
```
Comment (dnikolic98) [inline WeatherApp/WeatherApp/Feature/Components/WindWidgetView.swift]: degree

**Review by dnikolic98 — CHANGES_REQUESTED** [review]
For all the updated views please slim the bodies down, try to minimise padding modifiers and use the stacks spacing where possible. Keep in mind the newlines between the views. Long story short, take into consideration all the previous comments on other prs 😄

File: WeatherApp/WeatherApp/Feature/Components/WindWidgetView.swift (hunk 3 above)
Comment (SandroBoka) [reply WeatherApp/WeatherApp/Feature/Components/WindWidgetView.swift]: WindWidget is changed and fixed in the previous PR

**Pull request merged**